pkg/cli/pause_helm.go

```diff
@@ -30,6 +30,11 @@ func PauseHelm(ctx context.Context, globalFlags *flags.GlobalFlags, vClusterName
 		return err
 	}
 
+	if vCluster.IsSleeping() {
+		log.Infof("vcluster %s/%s is already paused using the platform driver", globalFlags.Namespace, vClusterName)
```

Comment:
Maybe reword this to `vcluster %s/%s is already paused by the platform`. Calling attention to the driver may be confusing and implies that they put it to sleep when the platform could have due to sleep mode configuration.